section4/exercises/burrito.rb

```diff
@@ -11,9 +11,25 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+  def add_toppings(item)
+    @toppings << item
+    puts "Your #{item} have been added"
+  end
+
+  def remove_toppings(item)
+     puts "Your #{item} have been removed"
```

Comment:
You have output that is is removed, but nothing has happened here to remove it!

Reply:
I added below def the following :: @toppings.delete(item)